scripts/install_third_party_libs_test.py

```diff
@@ -163,7 +163,7 @@ def mock_input(files, inplace):
         exists_swap = self.swap(os.path, 'exists', mock_exists)
         chdir_swap = self.swap(os, 'chdir', mock_chdir)
         mkdir_swap = self.swap(os, 'mkdir', mock_mkdir)
-        check_call_swap = self.swap(subprocess, 'check_call', mock_check_call)
+        check_call_swap = self.swap(subprocess, 'call', mock_call)
```

Comment:
Please rename this to `call_swap`.

Reply:
thanks for pointing this out, I'll make a follow up PR to resolve this issue later today. I'm going to merge this so that it doesn't affect new contributors.